### PR TITLE
Contact Page Updated

### PR DESCRIPTION
**Pull request by Abhishek971999**
Simple Contact Page Created!

### CODE REVIEW

File: frontend/src/components/contact/contact_us.js [hunk 1]
```diff
@@ -1,5 +1,65 @@
...
+  return (
+    <div class="contact-section">
+      <div class="contact-parent">
+        <div class="contact-child child1">
+          <img src="./images/contact.jpg" alt="" className="contact-image" />
+        </div>
 
-const ContactUs = () => <h1 className='contact-us'>CONTACT US</h1>;
+        <div class="contact-child child2">
+          <h3 class="contact-header-text">Get in touch</h3>
```
Comment (Kajol-Kumari) [inline frontend/src/components/contact/contact_us.js:13]: can we please have this heading in the center of the right sided form

File: frontend/src/components/contact/contact_us.js [hunk 2]
```diff
@@ -1,5 +1,65 @@
...
+                rows="4"
+                cols="20"
+                name="message"
+                Required="required"
+                placeholder="Your Message"
+              ></textarea>
+              <i className="far fa-comment-alt"></i>
+            </div>
+
+            <Button2 label="Submit" type="submit" />
```
Comment (Kajol-Kumari) [inline frontend/src/components/contact/contact_us.js:57]: Can we please have in the center of the right side form

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]

File: frontend/src/components/contact/contact_us.js (hunk 1 above)
Comment (Abhishek971999) [reply frontend/src/components/contact/contact_us.js:13]: Yes I have updated the requested changes in the recent commit.

File: frontend/src/components/contact/contact_us.js (hunk 2 above)
Comment (Abhishek971999) [reply frontend/src/components/contact/contact_us.js]: Yes I have updated the requested changes in the recent commit.

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :)

**Pull request merged**